TradeItIosTicketSDK2/TradeItRequestFactory.swift

```diff
@@ -31,7 +31,7 @@ class TradeItRequestFactory: NSObject {
     private static var envToHostDict: [TradeitEmsEnvironments: String] = [
         TradeItEmsProductionEnv : "https://ems.tradingticket.com/",
         TradeItEmsTestEnv: "https://ems.qa.tradingticket.com/",
-        TradeItEmsLocalEnv: "https://localhost:8443/"
+        TradeItEmsLocalEnv: "http://localhost:8080/"
```

Comment:
Is this change permanent/intentional?

Reply:
It was for testing locally, because I was unable to test it with https. If you know how to test with https locally, can you detail the procedure ?